README.md

```diff
@@ -1,22 +1,24 @@
-# iOS/macOS Swift SDK for LiveKit
+# Swift Multiplatform (iOS and macOS) SDK for LiveKit
 
-Official Client SDK for [LiveKit](https://github.com/livekit/livekit-server).
-Easily add video & audio capabilities to your iOS/macOS apps.
+This is the official LiveKit Swift client SDK. 
 
-## Docs & Example app
+Together with LiveKit [server](https://github.com/livekit/livekit-server), you can easily add real-time video/audio/data capabilities to your iOS or macOS apps.
 
-Docs and guides are at [https://docs.livekit.io](https://docs.livekit.io).
+<insert gif/image here />
```

Comment:
Placeholder for now ?

Reply:
Yeah, happy to use a video that you record =)